apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -419,7 +440,7 @@ def prepare_tag(
     # if the order of the blocks in the HTML is a correct solution, leave it unchanged, but if it
     # isn't we need to change it into a solution before displaying it as such
     data_copy = deepcopy(data)
-    data_copy["submitted_answers"] = {answer_name: correct_answers}
+    data_copy["submitted_answers"] = {answer_name: deepcopy(correct_answers)}
```

Comment:
This change was needed because the correct answers is getting passed to the grade function as a student submission, and we don't want any modifications that happen during that to persist to the actual saved correct answers.